### ISSUE TITLE
jx import --github --org fails panic: runtime error: invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by pasikon** [jenkins-x/jx#1104]
I tried to import a project using `import --github --org org1`, it detected repos, I selectrd desired one then what I got is:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x48 pc=0x17e9add]

goroutine 1 [running]:
github.com/jenkins-x/jx/pkg/jx/cmd.(*ImportOptions).getCurrentUser(0xc42063a000, 0xc420132180, 0xc420271568)
	/home/jenkins/go/src/github.com/jenkins-x/jx/pkg/jx/cmd/import.go:571 +0x2d
github.com/jenkins-x/jx/pkg/jx/cmd.(*ImportOptions).DraftCreate(0xc42063a000, 0x0, 0x0)
	/home/jenkins/go/src/github.com/jenkins-x/jx/pkg/jx/cmd/import.go:547 +0x627
github.com/jenkins-x/jx/pkg/jx/cmd.(*ImportOptions).Run(0xc42063a000, 0x1d74f09, 0x18)
	/home/jenkins/go/src/github.com/jenkins-x/jx/pkg/jx/cmd/import.go:322 +0x5cf
github.com/jenkins-x/jx/pkg/jx/cmd.(*ImportOptions).ImportProjectsFromGitHub(0xc4200d9d40, 0xc4206c0800, 0x2afb6e0)
	/home/jenkins/go/src/github.com/jenkins-x/jx/pkg/jx/cmd/import.go:387 +0x3f4
github.com/jenkins-x/jx/pkg/jx/cmd.(*ImportOptions).Run(0xc4200d9d40, 0xc420425ef0, 0x0)
	/home/jenkins/go/src/github.com/jenkins-x/jx/pkg/jx/cmd/import.go:279 +0x7fe
github.com/jenkins-x/jx/pkg/jx/cmd.NewCmdImport.func1(0xc4207e2a00, 0xc420425ef0, 0x0, 0x3)
	/home/jenkins/go/src/github.com/jenkins-x/jx/pkg/jx/cmd/import.go:141 +0x75
github.com/jenkins-x/jx/vendor/github.com/spf13/cobra.(*Command).execute(0xc4207e2a00, 0xc420425ec0, 0x3, 0x3, 0xc4207e2a00, 0xc420425ec0)
	/home/jenkins/go/src/github.com/jenkins-x/jx/vendor/github.com/spf13/cobra/command.go:766 +0x2c1
github.com/jenkins-x/jx/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc420152280, 0xc4206cd900, 0xc4207d1900, 0xc4206db900)
	/home/jenkins/go/src/github.com/jenkins-x/jx/vendor/github.com/spf13/cobra/command.go:852 +0x334
github.com/jenkins-x/jx/vendor/github.com/spf13/cobra.(*Command).Execute(0xc420152280, 0xc42000a0e0, 0x2ad4340)
	/home/jenkins/go/src/github.com/jenkins-x/jx/vendor/github.com/spf13/cobra/command.go:800 +0x2b
github.com/jenkins-x/jx/cmd/jx/app.Run(0xc4204aff70, 0x18a8bd9)
	/home/jenkins/go/src/github.com/jenkins-x/jx/cmd/jx/app/jx.go:18 +0x90
main.main()
	/home/jenkins/go/src/github.com/jenkins-x/jx/cmd/jx/jx.go:10 +0x22
```

My versions:

```
jx                 1.3.16
jenkins x platform 0.0.1466
kubernetes cluster v1.10.0
kubectl            v1.10.4
helm client        v2.9.1+g20adb27
helm server        v2.9.1+g20adb27
git                git version 2.17.1
```

**Comment by pasikon** [comment]
Succeeded when I did same with `--url`

**Comment by ccojocar** [comment]
@pasikon I tried to reproduce it with latest version of jx but couldn't generate the crash. Do you still get ti? Thanks

**Comment by olblak** [comment]
I also have the same issue when I run ``` jx import --github --org olblak --filter ldap``` 
NAME               VERSION
jx                              1.3.42
jenkins x platform    0.0.1602
kubernetes cluster   v1.10.3
kubectl                      v1.11.0
helm client                v2.7.2+g8478fb4
helm server              v2.7.2+g8478fb4
git                             git version 2.17.1

**Comment by pasikon** [comment]
@ccojocar I'm not able to redeploy the cluster right now but I will update you when I can

**Comment by angegar** [comment]
hi all,
i have the same kind of issues in running **jx compliance run**

` 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x1368fe9]

goroutine 1 [running]:
github.com/jenkins-x/jx/vendor/github.com/heptio/sonobuoy/pkg/client.(*SonobuoyClient).Run(0xc420551a70, 0xc420836ba0, 0x0, 0x0)
        /home/jenkins/go/src/github.com/jenkins-x/jx/vendor/github.com/heptio/sonobuoy/pkg/client/run.go:62 +0x2f9
github.com/jenkins-x/jx/pkg/jx/cmd.(*ComplianceRunOptions).Run(0xc420a0ea80, 0x0, 0x0)
        /home/jenkins/go/src/github.com/jenkins-x/jx/pkg/jx/cmd/compliance_run.go:65 +0x70
github.com/jenkins-x/jx/pkg/jx/cmd.NewCmdComplianceRun.func1(0xc420244280, 0x2ddc2b0, 0x0, 0x0)
        /home/jenkins/go/src/github.com/jenkins-x/jx/pkg/jx/cmd/compliance_run.go:49 +0x70
github.com/jenkins-x/jx/vendor/github.com/spf13/cobra.(*Command).execute(0xc420244280, 0x2ddc2b0, 0x0, 0x0, 0xc420244280, 0x2ddc2b0)
        /home/jenkins/go/src/github.com/jenkins-x/jx/vendor/github.com/spf13/cobra/command.go:766 +0x2c1
github.com/jenkins-x/jx/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc42037f180, 0xc420a0c500, 0xc4207f2000, 0xc420189b80)
        /home/jenkins/go/src/github.com/jenkins-x/jx/vendor/github.com/spf13/cobra/command.go:852 +0x30a
github.com/jenkins-x/jx/vendor/github.com/spf13/cobra.(*Command).Execute(0xc42037f180, 0xc42095a580, 0x2004c60)
        /home/jenkins/go/src/github.com/jenkins-x/jx/vendor/github.com/spf13/cobra/command.go:800 +0x2b
github.com/jenkins-x/jx/cmd/jx/app.Run(0xc4206f9f78, 0xc4200a4058)
        /home/jenkins/go/src/github.com/jenkins-x/jx/cmd/jx/app/jx.go:17 +0x90
main.main()
        /home/jenkins/go/src/github.com/jenkins-x/jx/cmd/jx/jx.go:10 +0x22`

**Comment by ccojocar** [comment]
@angegar This is fix in version `1.3.791`.  You can upgrade your jx with `jx upgrade cli`, `jx upgrade platform`. 

```
$jx compliance run
INFO[0000] created object                                name=heptio-sonobuoy namespace= resource=namespaces
INFO[0000] created object                                name=sonobuoy-serviceaccount namespace=heptio-sonobuoy resource=serviceaccounts
INFO[0000] object already exists                         name=sonobuoy-serviceaccount-heptio-sonobuoy namespace= resource=clusterrolebindings
INFO[0000] object already exists                         name=sonobuoy-serviceaccount namespace= resource=clusterroles
INFO[0000] created object                                name=sonobuoy-config-cm namespace=heptio-sonobuoy resource=configmaps
INFO[0000] created object                                name=sonobuoy-plugins-cm namespace=heptio-sonobuoy resource=configmaps
INFO[0000] created object                                name=sonobuoy namespace=heptio-sonobuoy resource=pods
INFO[0000] created object                                name=sonobuoy-master namespace=heptio-sonobuoy resource=services

$ jx version
Failed to get helm version: failed to run 'helm version --short' command in directory '', output: 'Error: unknown command "version --short" for "helm"
Run 'helm --help' for usage.': exit status 1
NAME               VERSION
jx                 1.3.79
jenkins x platform 0.0.1722
kubernetes cluster v1.10.5-gke.0
kubectl            v1.10.4
git                git version 2.15.2 (Apple Git-101.1)

$jx compliance status
Compliance tests are still running, it can take up to 60 minutes.%
```
